core/src/main/java/org/mule/exception/RollbackMessagingExceptionStrategy.java

```diff
@@ -18,7 +21,14 @@
 {
     private RedeliveryExceeded redeliveryExceeded;
     private Integer maxRedeliveryAttempts;
-    
+
+
+    @Override
+    protected void doInitialise(MuleContext muleContext) throws InitialisationException
+    {
+        LifecycleUtils.initialiseIfNeeded(redeliveryExceeded);
```

Comment:
This is the second time I see there is a explicit invocation to initialise a dependency. I understand the reasons of this, but this is somethings we must be sure to document very carefully to minize problems with user's custom code